src/monacoCompletionProvider.ts

```diff
@@ -62,7 +62,7 @@ declare module 'monaco-editor' {
 }
 
 export const OMonacoSite = {
-  UNSPECIFIED: 0,
```

Comment:
Sorry, to be more specific, let's keep UNSPECIFIED = 0 and add CUSTOM as a new value.

Reply:
No problem, I recovered it.